lighthouse-core/gather/computed/trace-of-tab.js

```diff
@@ -88,7 +88,10 @@ class TraceOfTab extends ComputedArtifact {
       firstMeaningfulPaint = lastCandidate;
     }
 
-    const onLoad = keyEvents.find(e => e.name === 'MarkLoad' && e.ts > navigationStart.ts);
+    const onLoad = frameEvents.find(e => e.name === 'loadEventStart' && e.ts > navigationStart.ts);
+    const domContentLoaded = frameEvents.find(
+      e => e.name === 'domContentLoadedEventStart' && e.ts > navigationStart.ts
```

Comment:
Actually......

The doc uses dclEnd rather than dclStart.
Since it's used as `fi = Math.max(DCL, FI)`... the End value seems much more accurate. (Handlers are setup during DCL, and the main thread is unavailable during this time)